wallet-new/demo/README.md

```diff
@@ -0,0 +1,76 @@
+# Demo
+
+## Getting Started
+
+This module provides an executable for starting a demo cluster of wallet nodes.
+It is designed to remove all the overhead of setting up a configuration
+and an environment and to _just work_, out-of-the-box. Minor configuration
+adjustments are however possible via environment variables. 
+
+```
+stack test cardano-sl-wallet-new:demo
+```
+
+This spawns three wallet-nodes running respectively on:
+
+| NodeId | Address        | API Address    | API Doc Address |
+| ---    | ---            | ---            | ---             |
+| node0  | localhost:3000 | localhost:8090 | localhost:8190  |
+| node1  | localhost:3001 | localhost:8091 | localhost:8191  |
+| node2  | localhost:3002 | localhost:8092 | localhost:8192  |
```

Comment:
TODO: Fix the documentation here, we now start 4 core nodes, 1 relay and 1 wallet